Source/CarthageKit/Project.swift

```diff
@@ -522,7 +522,7 @@ public final class Project {
 			})
 	}
 	
-	public func buildOrderForResolvedCartfile(cartfile: ResolvedCartfile) -> SignalProducer<Dependency<PinnedVersion>, CarthageError> {
+	public func buildOrderForResolvedCartfile(cartfile: ResolvedCartfile, dependenciesToInclude: [String]? = nil) -> SignalProducer<Dependency<PinnedVersion>, CarthageError> {
```

Comment:
I think an optional here makes more sense than using an empty list to represent "include all dependencies". But I can see how an empty list is closer to how the command line arguments work.